app/controllers/miq_ae_tools_controller.rb

```diff
@@ -395,7 +395,7 @@ def get_form_vars
       @resolve[:new][:target_class] = params[:target_class]
       klass = CustomButton.name_to_button_class(params[:target_class])
       unless klass.nil?
-        targets = klass.all
+        targets = klass
```

Comment:
This line is unnecessary, we could just be using `klass` on line 399.

However, this highlights a different issue, and that is that the `name_to_button_class` method no longer does what it says it does. I did a search and the above line (396) is the only place that is using the `name_to_button_class` method, so maybe we should change the name to something like `name_to_button_class_targets` or something similar. Then instead of using `klass` on line 396, you would use `targets`.

@gmcculloug Do you by chance know if that `name_to_button_class` method is ever called dynamically?